crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -631,6 +646,22 @@ impl TurboJson {
             }),
         );
     }
+
+    /// Adds a sibling relationship from task to sibling
+    pub fn with_sibling(&mut self, task: TaskName<'static>, sibling: &TaskName) {
+        if self.extends.is_empty() {
```

Comment:
What's the reason for this? Do we only add siblings to workspace configs?

Reply:
At this moment, yes.

Reply:
Could we document this then?